esignet-default.properties

```diff
@@ -130,14 +130,18 @@ mosip.esignet.integration.binding-validator=BindingValidatorServiceImpl
 mosip.esignet.integration.authenticator=MockAuthenticationService
 mosip.esignet.integration.key-binder=IdaKeyBinderImpl
 mosip.esignet.integration.audit-plugin=IdaAuditPluginImpl
-mosip.esignet.integration.captcha-validator=GoogleRecaptchaValidatorService
 mosip.esignet.integration.vci-plugin=IdaVCIssuancePluginImpl
 
 # captcha validator
 mosip.esignet.captcha.required=
 mosip.esignet.captcha.validator-url=https://${mosip.api.internal.host}/v1/captcha/validatecaptcha
 mosip.esignet.captcha.module-name=esignet
 mosip.esignet.captcha.site-key=${esignet.captcha.site.key}
+mosip.esignet.captcha-validator.secret=${esignet.captcha.secret.key}
```

Comment:
This is not required, as its part of captcha-validation-service based on modulename

Reply:
removed